app/services/doc_auth/acuant/responses/get_results_response.rb

```diff
@@ -97,12 +97,19 @@ def processed_alerts
 
         def log_alerts(alerts)
           log_alert_results = {}
+          has_failed = {}
+
           alerts.keys.each do |key|
             alerts[key.to_sym].each do |alert|
-              side = alert[:side] || 'no_side'
-              log_alert_results[alert[:name].
+              alert_name_key = alert[:name].
                 downcase.
-                parameterize(separator: '_').to_sym] = { "#{side}": alert[:result] }
+                parameterize(separator: '_')
+              side = alert[:side] || 'no_side'
+              unless has_failed[alert_name_key.to_sym]
+                log_alert_results[alert_name_key.to_sym] =
+                  { "#{side}": alert[:result] }
+              end
```

Comment:
maybe similar to doug's comment, but maybe just want to warn/check on if we're overwriting? Like no need to keep track of a `has_failed` hash separately?

```suggestion
              side = alert[:side] || 'no_side'
              if log_alert_results[alert_name_key.to_sym][side.to_sym]
                # warn
              end
              log_alert_results[alert_name_key.to_sym][side.to_sym] = alert[:result]
```